spec/models/form_profile_spec.rb

```diff
@@ -52,10 +52,10 @@
     {
       'street' => street_check[:street],
       'street2' => street_check[:street2],
-      'city' => user.va_profile[:address][:city],
-      'state' => user.va_profile[:address][:state],
-      'country' => user.va_profile[:address][:country],
-      'postal_code' => user.va_profile[:address][:postal_code][0..4]
+      'city' => user.address.city,
+      'state' => user.address.state,
+      'country' => user.address.country,
+      'postal_code' => user.address.postal_code[0..4]
```

Comment:
```suggestion
      'postal_code' => user.address.postal_code[0...5]
```

I'm not sure which is more intuitive. One has to be expecting zero-index here. Maybe my programmer brain is on wrong but I was looking for suspiciously at the `4`.

You could also do:

```suggestion
      'postal_code' => user.address.postal_code.slice(0, 5)
```